src/pages/Interactions/index.mdx

```diff
@@ -0,0 +1,44 @@
+## Process Flow
+
+Below is a comprehensive depiction, following a member's journey from induction, showcasing the various stages they undergo within the fellowship.
+
+
+### **Becoming a Polkadot Technical Fellowship Member**
+
+A non-member can become ranked by the felowship through `pallet_ranked_collective` or `pallet_core_fellowship` respectively;
+
+**Ranked Collective**
+
+To get added as a member and promoted to **"Rank 1"**, an existing member of the fellowship needs to submit a referendum with the preimage of a batch call that has `fellowshipCollective.addMember` and
+`fellowshipCollective.promoteMember` on "2/Proficients" track. Provied the requirements for member has been met as stated in the manifesto.
+
+On Polkadot-JS UI for Polkadot Collectives, navigate to Governance > Fellowship > Referenda and click on "Add Preimage". This preimage can be submitted by anyone.
+
+<img className={"adj-img"} src={preimage} alt="preimage" />
```

Comment:
Where is this image imported from?

Reply:
This content is to be removed in favour of [this PR](https://github.com/polkadot-fellows/dashboard/pull/132).